app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
```

Comment:
- [ ] According to the project requirements, the log in and sign up page should follow the given wireframe.
### Required
![image](https://github.com/Chimoneg27/Rails-Blog-App/assets/45460192/55b48ff7-02bf-4052-80ed-e355c8c779f3)
### Your Implementation
![image](https://github.com/Chimoneg27/Rails-Blog-App/assets/45460192/14e9bd17-c724-4832-ab06-b46f2b26f7c4)